### PR TITLE
Improve utils docs

### PR DESCRIPTION
**Pull request by Vultraz**
Have had this sitting around locally for a couple months. Wanted to add more, but I might as well add what I have.

### CODE REVIEW

File: packages/utils/src/index.js [hunk 1]
```diff
@@ -201,18 +222,22 @@ export const captureSuspensePromises = executors => {
...
  * Compares two version strings or number arrays in the major.minor.patch format.
- * Returns true if comp if each element of comp is greater than than base.
+ * @param {Array<Number>|String} comp The version to compare.
+ * @param {Array<Number>|String} base The version against which to compare.
+ * @returns True if each element of comp is greater than or equal base.
  */
 export const versionGreaterOrEqualTo = (comp, base) => {
   const parseVersion = input => {
-    if (typeof input === 'object') {
+    if (input instanceof Array) {
```
Comment (Vultraz) [inline packages/utils/src/index.js:240]: Tiny random change. When I originally wrote this function I didn't know about `instanceof` and I had intended this check to be for arrays.

Comment (pahaz) [reply packages/utils/src/index.js:240]: It's better to use `Array.isArray()`. instanceof with Arrays produce some problems ...

Comment (Vultraz) [reply packages/utils/src/index.js:240]: Do Javascript’s quirks never end ☹️

Comment (pahaz) [reply packages/utils/src/index.js:240]: I can create PR. Because it's may produce problems for me.

**Pull request merged**